go.mod

```diff
@@ -20,7 +20,7 @@ replace (
 require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/cilium/charts v0.0.0-20230628232147-9992996cd311
-	github.com/cilium/cilium v1.14.0-rc.0
+	github.com/cilium/cilium v1.14.0-snapshot.5
```

Comment:
For those of you wondering, `snapshot` is semantically higher than `rc`. We'll switch to using `pre` in the Cilium v1.15 release cycles. To avoid problems, RCs are dual tagged with `snapshot`. In this specific instance, `-snapshot.5` is the same as `-rc.0`.

Reply:
Thank you for the explanation :heart: TIL